README.md

```diff
@@ -147,25 +149,25 @@ The system described in this paper wants to provide several guarantees for the e
+- *Bob* verifies that the access token is valid, and starts a **handshake** to create a shared *session key* with *Alice*.
+- *Alice* participates in the **handshake**.
+- *Alice* and *Bob* now have the same *session key*.
+- *Bob* encrypts *X* using the previously calculated *session key* and shares it with *Alice*, either on- or off-chain.
 
-If both *Alice* and *Bob* played by the rules, everyone is happy and no conflict resolution is needed. But what happens when one of them acts maliciously? *Alice*, once she obtained the private asset *X*, can say it was not the asset she paid for, and ask for a refund. On the other side, *Bob* can transfer to *Alice* total garbage.
+If both *Alice* and *Bob* played by the rules, everyone is happy and no conflict resolution is needed. But what happens if one of them acts maliciously? *Alice*, once she obtained the private asset *X*, could say it was not the asset she paid for, and ask for a refund. On the other side, *Bob* could send *Alice* total garbage instead of *X*.
 
-The important part is the **handshake** between *Alice* and *Bob*. Here the blockchain plays a crucial role. By storing all those steps as transactions in the system, *Alice* or *Bob* are able to expose malicious behaviour to a third party.
+The important part is the **handshake** between *Alice* and *Bob*. Here the blockchain plays a crucial role. By storing all those steps as transactions in the system, *Alice* or *Bob* could expose malicious behaviour to a third party.
 
-Alas, finding the culprit is not enough, especially when they are anonymous. To properly disincentivize malicious actors, a **stake** is added to the **handshake**. The stake can be kept secret between parties, and revealed when a conflict occurs. By using [crypto conditions][cc:draft] combinatory logic, *Alice* and *Bob* can create a threshold condition `n-of-m`, where `m` defines the third parties involved that can eventually release the funds to the injured party.
+Alas, finding the culprit is not enough, especially when they are anonymous. To properly deter malicious actors, a **stake** is added to the **handshake**. The stake can be kept secret between parties, and revealed when a conflict occurs. By using [crypto conditions][cc:draft] combinatory logic, *Alice* and *Bob* can create a threshold condition `n-of-m`, where `m` defines the third parties involved that can eventually release the funds to the injured party.
 
-**Note: stake is not a requirement, but it adds another level of security. If the identity of the parties is known (i.e. KYC has been done correcty), then the litigation can be solved in other ways, by legal means for example.**
+**Note: a stake is not a requirement, but it adds another level of protection to the transaction. If the identity of the parties is known, then the dispute can be resolved in other ways, like the legal system. However, this may be less efficient than stake-based resolution.**
```

Comment:
I would use "such as" rather than "like" in the sentence:

"If the identity of the parties is known, then the dispute can be resolved in other ways, like the legal system."